app/src/main/java/se/inera/intyg/minaintyg/util/html/HTMLFactory.java

```diff
@@ -0,0 +1,49 @@
+package se.inera.intyg.minaintyg.util.html;
+
+public class HTMLFactory {
```

Comment:
Add private constructors and throw new IllegalStateException("Utility class"); if used.